docs/install/installing-istio.md

```diff
@@ -69,6 +69,18 @@ without automatic sidecar injection.
    Wait a few seconds for the CRDs to be committed in the Kubernetes API-server,
    then continue with these instructions.
 
+1. Create `istio-system` namespace
+    ```shell
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: istio-system
+      labels:
+        istio-injection: disabled
+    EOF
+    ```
```

Comment:
Right idea -- likely Ok, but when I copy & paste from this PR, zsh is unhappy w/ it.

Reply:
Could you please share any error information about this? This piece works well in my side.

Reply:
The leading spaces was causing failure.  It went into "pipe heredoc>" mode rather than ending.

Reply:
I think if you copy it from the doc directly (https://github.com/knative/docs/blob/master/docs/install/installing-istio.md#downloading-istio-and-installing-crds), there should be leading spaces.